src/views/tutorial-view/components/image-config/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactElement, ReactNode } from 'react'
+import getValidatedImgChild from './get-validated-image-child'
+import MdxImg from '../mdx-img'
+// import classNames from 'classnames'
+import s from './image-config.module.css'
+
+/**
+ * Configure extra settings for images via MDX.
+ * Designed to wrap a single markdown image.
+ */
+export default function ImageConfig({
```

Comment:
This component was largely ported over from Learn - [original component](https://github.com/hashicorp/learn/tree/master/components/image-config).